src/routes/boot.ts

```diff
@@ -462,11 +465,15 @@ export const getBootData = async (
   ) {
     return loggedInBoot(con, req, res, false, middleware);
   }
+
   const whoami = await dispatchWhoami(req);
   if (whoami.valid) {
     if (whoami.cookie) {
       setRawCookie(res, whoami.cookie);
     }
```

Comment:
Should we set cookie before we know if user is verified, I am not sure for what is it for?

Reply:
Yeah verified doesn't mean your not logged in basically.
It's just that you still need to verify your email if that makes sense.

Reply:
👍